obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+  status: 1
+- agasc_id:
+  - 397935912
+  comments: Never acquired
+  mp_starcat_time: 2017:068:13:20:24.092
+  status: 1
+- agasc_id:
+  - 991169936
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 991170368
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 991175320
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 991559968
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 991560432
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 991560472
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 991563000
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 991564392
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 798238560
+  comments: Tracking affected by imposter
+  mp_starcat_time: 2017:075:03:00:23.891
+  status: 1
+- agasc_id:
+  - 1200756680
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 1200757192
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 1200758760
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 1200760024
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 1200888464
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 45616472
+  comments: Faint star affected by imposter. Set mag to MAXMAG
+  mp_starcat_time: 2017:119:02:36:19.023
+  status: 1
+- agasc_id:
+  - 311824376
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2017:198:09:20:24.644
+  status: 1
+- agasc_id:
+  - 913441888
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2017:259:02:11:04.769
+  status: 1
+- agasc_id:
+  - 377102496
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2017:314:20:46:46.000
+  status: 1
+- agasc_id:
+  - 92812824
+  comments: Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2017:354:04:35:24.949
+  status: 1
+- agasc_id:
+  - 653529952
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2018:002:23:37:48.551
+  status: 1
+- agasc_id:
+  - 123874048
+  comments: Bright star
+  mp_starcat_time: 2018:014:04:03:16.301
+  status: 1
+- agasc_id:
+  - 797847184
+  comments: Tracking affected by imposter. Set mag to 12.3 +- 0.2
+  mp_starcat_time: 2018:068:11:54:25.784
+  status: 1
+- agasc_id:
+  - 1105068512
+  comments: spoiler 22 arcsec away
```

Comment:
I suspect this is another high proper motion star http://simbad.u-strasbg.fr/simbad/sim-id?Ident=%403436260&Name=HD%20204961&submit=submit

Reply:
I put this in the bad list as a high proper motion star